.github/workflows/README.md

```diff
@@ -8,7 +8,7 @@ Accessing them can be done via the vars object.
 | Variable       | Description                                             | Example value |
 | -------------- | ------------------------------------------------------- | ------------- |
 | DOTNET_VERSION | The version of dotnet to load with actions/setup-dotnet | 6.0.x         |
-| NODE_VERSION   | The version of node.js to load with actions/setup-node  | 18.x          |
+| NODE_VERSION   | The version of node.js to load with actions/setup-node  | 20.x          |
```

Comment:
22?

Reply:
It was meant to be 22. Good catch, let me fix that.